<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataView</class>
 <widget class="QDialog" name="DataView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    font: bold;
   border: 2px solid rgb(100, 100, 100);
    border-radius: 9px;
   margin-top: 6px;
background-color: #E0E0E0
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
     margin-left: auto;
     margin-right: auto;
}

</string>
     </property>
     <property name="title">
      <string>Monitoring</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_tmpN">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Température:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_tmp">
                <property name="font">
                 <font>
                  <family>Quicksand Medium</family>
                  <pointsize>50</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLabel {  color : green; }</string>
                </property>
                <property name="text">
                 <string>00.0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_humN">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Humidité:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_hum">
                <property name="font">
                 <font>
                  <family>Quicksand Medium</family>
                  <pointsize>50</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLabel {color : blue; }</string>
                </property>
                <property name="text">
                 <string>00.0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_tmp_s">
                <property name="font">
                 <font>
                  <family>URW Palladio L</family>
                  <pointsize>50</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLabel {color : green; }</string>
                </property>
                <property name="text">
                 <string>°C</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_hum_s">
                <property name="font">
                 <font>
                  <family>URW Bookman L</family>
                  <pointsize>50</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLabel {  color : blue; }</string>
                </property>
                <property name="text">
                 <string>%</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCustomPlot" name="customPlot" native="true">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
